Networking/Networking/Remote/TelemetryRemote.swift

```diff
@@ -16,7 +16,12 @@ public class TelemetryRemote: Remote {
     public func sendTelemetry(for siteID: Int64, versionString: String, completion: @escaping (Result<Void, Error>) -> Void) {
         let path = "tracker"
         let parameters = ["platform": "ios", "version": versionString]
-        let request = JetpackRequest(wooApiVersion: .wcTelemetry, method: .post, siteID: siteID, path: path, parameters: parameters)
+        let request = JetpackRequest(wooApiVersion: .wcTelemetry,
+                                     method: .post,
+                                     siteID: siteID,
+                                     path: path,
+                                     parameters: parameters,
+                                     availableAsRESTRequest: true)
```

Comment:
Do we want to also add test cases for `TelemetryRemoteTests`?

Reply:
Added a new case in 3c3e415eb9aa34eabd53275cb1ab2a764a17423a.